obfuscator-core/src/main/java/me/superblaubeere27/jobf/processors/name/NameObfuscation.java

```diff
@@ -291,7 +291,7 @@ private boolean isClassExcluded(ClassWrapper classWrapper) {
     }
 
     private boolean isMethodExcluded(String owner, MethodWrapper methodWrapper) {
-        String str = owner + '.' + methodWrapper.originalName;
+        String str = owner + '/' + methodWrapper.originalName;
```

Comment:
It is intended that it looks like `package/Class.method` and not like `package/Class/method`. It is personal preference. If there is any problem with it, I will change it.

Reply:
Look at the ```compileExcludePattern``` method. They do not determine what the method or field will change, it just changes all the points to slashes. As a result, an incorrect pattern is formed.